src/styles/app-theme/app.typography.js

```diff
@@ -45,6 +45,12 @@ const appTypography = {
     lineHeight: '20px',
     letterSpacing: '0.1px'
   },
+  subtitle3: {
```

Comment:
it looks almost the same as 'button' do we really need it?

Reply:
fixed